jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -20,7 +20,7 @@
 
     public static SchemaMetadata readRemoteMetadata(final String connectionString, final String user,
             final String password, String catalog, String schema, final List<String> tableFilter,
-            final String dialectName, final JdbcAdapterProperties.ExceptionHandlingMode exceptionMode)
+            final String dialectName, final JdbcAdapterProperties.ExceptionHandlingMode exceptionMode, final List<String> ignoreErrorList)
             throws SQLException, AdapterException {
         assert (catalog != null);
```

Comment:
Use Exception with proper user error message.

Reply:
leave that to a refactoring branch if we want to remove all asserts